templates/AddNewCoreLibrary.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+    .SYNOPSIS
+    Adds the class library and NuGet projects necessary to support a new core library
```

Comment:
Definitely here. 
And yes, something in contributor.md sounds like it would be a good idea as I take it we expect others to need this periodically.